RevenueCatUI/Data/Localization.swift

```diff
@@ -164,6 +164,8 @@ private extension Localization {
         switch locale.languageCodeIdentifier {
         // Abbreviated does not fully work with Japanese
         case "ja": return [.brief]
+        // Abbreviated is too short for German
+        case "de": return [.full, .brief]
```

Comment:
Can you provide more context for this change? Thanks!

Reply:
This change is for the variable {{ price_per_period }} and {{ total_price_and_per_month }} in Paywalls

Weekly Before `9,99 €/Wo.` Now `no change`
Monthly Before `9,99 €/M` Now `9,99 €/Mon.`
Yearly Before `9,99 €/J` Now `no change`